files/build_templates/sonic_debian_extension.j2

```diff
@@ -246,6 +246,16 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-utilities-data_*.deb || \
 # in bash.bashrc, so we copy a version of the file with it enabled here.
 sudo cp -f $IMAGE_CONFIGS/bash/bash.bashrc $FILESYSTEM_ROOT/etc/
 
+# Remove 'PyGObject' package installed via apt
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y remove python3-gi
+
+# Install prerequisites needed for installing the dependent Python packages of sonic-host-services
+# These packages can be uninstalled after installation
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install libcairo2-dev libdbus-1-dev libgirepository1.0-dev libsystemd-dev pkg-config
```

Comment:
>libcairo2-dev [](start = 86, length = 13)

Why need `-dev` packages? They are only needed in build time.

Reply:
The '-dev' packages are used for building the dependent Python packages (PyGObject, dbus-python, systemd-python) while installing sonic-host-services via pip and are uninstalled after sonic-host-services is installed.